dts/riscv/it8xxx2.dtsi

```diff
@@ -240,7 +240,16 @@
 			interrupts = <39 IRQ_TYPE_EDGE_RISING>;
 			interrupt-parent = <&intc>;
 		};
-		timer: timer@f01f00 {
+
+		twd0: watchdog@f01f00 {
+			compatible = "ite,it8xxx2-watchdog";
+			reg = <0x00f01f00 0x0062>;
+			label = "TWD_0";
+			interrupts = <30 IRQ_TYPE_EDGE_RISING>;
+			interrupt-parent = <&intc>;
+		};
+
+		timer: timer@f01f10 {
```

Comment:
If you update this, please also update the 'reg' property below

Reply:
Yes, I will.